tests/simple/testdata/proto3.textproto

```diff
@@ -1,8 +1,436 @@
+    expr: "TestAllTypes{single_uint32: 1u}"
+    value {
+      object_value: {
+        [type.googleapis.com/google.api.expr.test.v1.proto3.TestAllTypes]
+        { single_uint32: 1 }
+      }
+    }
+  }
+
+  test {
+    name: "uint64"
+    container: "google.api.expr.test.v1.proto3"
+    expr: "TestAllTypes{single_uint64: 9999u}"
+    value {
+      object_value {
+        [type.googleapis.com/google.api.expr.test.v1.proto3.TestAllTypes]
+        { single_uint64: 9999 }
+      }
+    }
+  }
+
+  test {
+    name: "sint32"
+    container: "google.api.expr.test.v1.proto3"
+    expr: "TestAllTypes{single_sint32: -3}"
+    value {
+      object_value {
+        [type.googleapis.com/google.api.expr.test.v1.proto3.TestAllTypes]
+        { single_sint32: -3 }
+      }
+    }
+  }
+
+  test {
+    name: "sint64"
+    container: "google.api.expr.test.v1.proto3"
+    expr: "TestAllTypes{single_sint64: 255}"
+    value {
+      object_value {
+        [type.googleapis.com/google.api.expr.test.v1.proto3.TestAllTypes]
+        { single_sint64: 255 }
+      }
+    }
+  }
+
+  test {
+    name: "fixed32"
+    container: "google.api.expr.test.v1.proto3"
+    expr: "TestAllTypes{single_fixed32: 43u}"
+    value {
+      object_value {
+        [type.googleapis.com/google.api.expr.test.v1.proto3.TestAllTypes]
+        { single_fixed32: 43 }
+      }
+    }
+  }
+
+  test {
+    name: "fixed64"
+    container: "google.api.expr.test.v1.proto3"
+    expr: "TestAllTypes{single_fixed64: 1880u}"
+    value {
+      object_value {
+        [type.googleapis.com/google.api.expr.test.v1.proto3.TestAllTypes]
+        { single_fixed64: 1880 }
+      }
+    }
+  }
+
+  test {
+    name: "sfixed32"
+    container: "google.api.expr.test.v1.proto3"
+    expr: "TestAllTypes{single_sfixed32: -404}"
+    value {
+      object_value {
+        [type.googleapis.com/google.api.expr.test.v1.proto3.TestAllTypes]
+        { single_sfixed32: -404 }
+      }
+    }
+  }
+
+  test {
+    name: "sfixed64"
+    container: "google.api.expr.test.v1.proto3"
+    expr: "TestAllTypes{single_sfixed64: -1}"
+    value {
+      object_value {
+        [type.googleapis.com/google.api.expr.test.v1.proto3.TestAllTypes]
+        { single_sfixed64: -1 }
+      }
+    }
+  }
+
+  test {
+    name: "float"
+    container: "google.api.expr.test.v1.proto3"
+    expr: "TestAllTypes{single_float: 3.1416}"
+    value {
+      object_value {
+        [type.googleapis.com/google.api.expr.test.v1.proto3.TestAllTypes]
+        { single_float: 3.1416 }
+      }
+    }
+  }
+
+  test {
+    name: "double"
+    container: "google.api.expr.test.v1.proto3"
+    expr: "TestAllTypes{single_double: 6.022e23}"
+    value {
+      object_value {
+        [type.googleapis.com/google.api.expr.test.v1.proto3.TestAllTypes]
+        { single_double: 6.022e23 }
+      }
+    }
+  }
+
+  test {
+    name: "bool"
+    container: "google.api.expr.test.v1.proto3"
+    expr: "TestAllTypes{single_bool: true}"
```

Comment:
Since 'false' is the default value, does setting `false` return an empty `TestAllTypes` value?

Reply:
Will test this in a future PR.